Userland/Libraries/LibJS/Lexer.cpp

```diff
@@ -399,12 +399,11 @@ Token Lexer::next()
                 do {
                     consume();
                 } while (!is_eof() && !is_block_comment_end());
-                if (is_eof())
-                    unterminated_comment = true;
-                consume(); // consume *
-                if (is_eof())
+                if (!is_eof()) {
+                    consume();
+                    consume();
+                } else
                     unterminated_comment = true;
-                consume(); // consume /
```

Comment:
Please add the comments back, they're useful

Reply:
Will do (don't have time right now, but will later)

Reply:
Fun fact: I forgot I had this in here! I think it might slightly improve performance in that it skips a branch, but I have not actually tested this, so this might actually be causing the test to fail....